chef/cookbooks/neutron/recipes/network_agents.rb

```diff
@@ -74,6 +74,18 @@
   subscribes :run, resources(cookbook_file: enable_ip_forward_file), :delayed
 end
 
+neighbour_table_overflow_file = "/etc/sysctl.d/50-neutron-neighbour-table-overflow.conf"
+cookbook_file neighbour_table_overflow_file do
+  source "sysctl-neighbour-table-overflow.conf"
+  mode "0644"
+end
+
+bash "reload neighbour-table-overflow.conf" do
+  code "/sbin/sysctl -e -q -p #{neighbour_table_overflow_file}"
+  action :nothing
+  subscribes :run, resources(cookbook_file: neighbour_table_overflow_file), :delayed
+end
```

Comment:
Don't we also want that on compute nodes? (in case DVR is used)

So that'd be more like in `common_agent.rb`.